app/src/main/java/org/openimis/imispolicies/AppInformation.java

```diff
@@ -1,10 +1,12 @@
 package org.openimis.imispolicies;
+import static org.openimis.imispolicies.BuildConfig.API_BASE_URL;
+import static org.openimis.imispolicies.BuildConfig.SHOW_CONTROL_NUMBER_MENU;
 
  public class AppInformation {
      public static class DomainInfo {
 
 //         URL address of openIMIS demo server => please update with your own Web Services base URL/IP
-         private static String _Domain = "https://demo.openimis.org/rest/";
+         private static String _Domain = API_BASE_URL;
 
          private static final String DEFAULT_RAR_PASSWORD = ")(#$1HsD";
```

Comment:
Could we add this one on the flavor config list ?

Reply:
Good catch. I'll do this.